### ISSUE TITLE
7.0.2 Road Tool skill requirement repair issue

### ISSUE THREAD

**Issue by Senlarus** [StrangeLoopGames/EcoIssues#3958]
The road tool requires road construction 1 to build it, but requires wood construction 1 to repair it. I don't think this was your intent with this tool. It should probably require road construction 1 to repair instead.

![image](https://user-images.githubusercontent.com/1503963/35839291-a26f2030-0abd-11e8-94c1-36cf38de3708.png)

**Comment by Senlarus** [comment]
You sure this isn't a bug? The road tool was recently updated to road construction 1 to craft in one of the staging builds. Before that, while still in staging, it required wood construction 1 to craft.

**Comment by craig-jong** [comment]
AUTOGEEEEEEEEEEEEEEEEEEEEEEEEEEEEEN

**Comment by craig-jong** [comment]
*will fix

**Comment by LiaungYip** [comment]
Fixed in 7.0.6.

```
[...snip...]
Changelog part 2 (too big for Discord):
- Fixed inventory synchronization when rapidly placing blocks
- Fixed an issue with the tailings cache that could cause inordinate amounts of pollution to be spread.
- Road Tool repair skill changed from Wood Construction to Road Construction
```

**Comment by MarjieAVolk** [comment]
Thank you, haha
